generator/cli/index.ts

```diff
@@ -64,7 +66,11 @@ async function main(): Promise<void> {
   }
 
   if (args['--generate-type-script']) {
-    generateTypeScript({ templatesRepoDir, outputDir: generatedDir })
+    await generateTypeScript({
+      templatesRepoDir,
+      outputDir: generatedDir,
+      prettier: args['--prettier'] ?? false,
```

Comment:
I tried a quick pass at integrating prettier but it was throwing a runtime error I didn't understand. So left this is in as opt-in for now.